src/routes/index.ts

```diff
@@ -587,6 +587,24 @@ router.route(config.UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_URL)
   )
   .all(updateManageTrustsOrchestrator.handler);
 
+router.route(config.UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_CHANGE_HANDLER_URL + config.TRUST_ID + '?')
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_UPDATE_MANAGE_TRUSTS),
+    authentication,
+    companyAuthentication,
+    navigation.hasTrustWithIdUpdate
+  )
+  .all(updateManageTrustsOrchestrator.trustChangeHandler);
+
+router.route(config.UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_CHANGE_HANDLER_URL + config.TRUST_ID + '?' + config.TRUSTEE_TYPE + '?' + config.TRUSTEE_ID + '?')
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_UPDATE_MANAGE_TRUSTS),
+    authentication,
+    companyAuthentication,
```

Comment:
please add hasTrustWithIdUpdate to this route

Reply:
The check for the trust is covered in the hasTrusteeWithId middleware below

Reply:
done withing hasTrusteeWithIdUpdate